alma/controllers/front/subscription.php

```diff
@@ -53,13 +63,22 @@ public function __construct()
     {
         parent::__construct();
         $this->context = Context::getContext();
-        $this->subscriptionHelper = new SubscriptionHelper(new AlmaInsuranceProductRepository());
-        $this->insuranceApiService = new InsuranceApiService();
+        $this->almaInsuranceProductRepository = new AlmaInsuranceProductRepository();
```

Comment:
$this->almaInsuranceProductRepository not use outside construct, you can set a simple variable